test/setup.js

```diff
@@ -9,6 +9,6 @@ afterEach(() => {
   // don't get intermittents from one test's mocks affecting another test's mocks.
   //
   // See https://github.com/facebook/jest/issues/7654
-  jest.resetAllMocks();
+  jest.restoreAllMocks();
```

Comment:
nit: it would be good to briefly explain why we chose `restoreAllMocks` instead of `resetAllMocks` or `clearAllMocks` in the comment for future proofing.

Reply:
Thanks!
I added this comment:
```
  // We use `restoreAllMocks` over `resetAllMocks` because this restores the mocks to
  // their initial value. `resetAllMocks` would keep the mock and make them
  // return `undefined`.
```